documentation/src/main/asciidoc/topics/ref_indexing_configuration.adoc

```diff
@@ -42,15 +42,19 @@ When setting a custom value, ensure that there are no conflicts between caches u
 == Index startup mode
 
 When {brandname} starts caches it can perform operations to ensure the index is consistent with data in the cache.
-By default no indexing operation takes place when a cache starts but you can configure {brandname} to:
+By default it:
+
+* Automatically clear or reindex the cache.
+** If data is volatile and the index is persistent then {brandname} clears the cache when it starts.
```

Comment:
`clears the cache when it starts` -> do you mean clear the indexes? :) I hope it doesn't clear the cache.